### ISSUE TITLE
BigQuery Schema page cuts off table edge

### ISSUE THREAD

**Issue by mtlynch** [m-lab/m-lab.github.io#203]
When the browser window is resized to be a narrower width, the rightmost column of the schema table gets cut off:

![image](https://cloud.githubusercontent.com/assets/7783288/15685760/2f134006-273b-11e6-961e-2caeea8c26ad.png)

Can we fix it so it breaks the text and resizes properly?

**Comment by shredtechular** [comment]
Tables in general always cause issues when viewing responsively.  There is a plethora of solutions out there with some that are fancier and more complex with 3rd party js libraries, etc.

Back in PR #173 we came to some conclusions to use Bootstrap's native solution that enables a horizontal scrollbar for users to be able to scroll right on the table when the table could no longer break down and fit within the user's viewport.

We had another solution in PR #159 that shrunk the table pretty well to 3 columns even on mobile devices, however, in order to do that we needed to force break long strings in the middle of a word.  Sometimes that caused some un-intended behavior if it broke a line in the middle of a word where it shouldn't.

Those were the easier solutions for the bigger data tables that are on the site, there are other options, but they could get tricky and maybe overly complex when thinking about creating/editing markdown tables, or require the need for additional 3rd party js also outlined in PR #159.
